### ISSUE TITLE
Add a recycling box for glass bottles

### ISSUE THREAD

**Issue by ajlennon** [DoESLiverpool/somebody-should#836]
Assuming we don't have one (can't see anything in the kichette can we get another recycling box for glass bottles as we are seeing a build of of them around and about.

**Comment by jackie1050** [comment]
Could we get a cardboard box for this and try it out for a while? I'd be happy to take part in a rota (once I get my car back ...)

**Comment by ajlennon** [comment]
Yeah I don't mind taking them to the bottle bank every now and again. 

Plus at present well meaning people are leaving glass bottles all over the place in cubby holes because (imho) they have a feeling they are expected to be recycled but can't work out where to put them.

So if nothing else it'll make things a bit tidier...

**Comment by ajlennon** [comment]
Where do the recycling boxes come from? Can somebody order one?

**Comment by johnmckerrell** [comment]
The boxes we have were provided by Cara/Bulky Bobs. I suspect we need something stronger for glass though. We did have a plastic box but it was falling apart so we dumped it during the move.

Are you including milk bottles in this? Should we actually be giving them back to the milk man?

**Comment by ajlennon** [comment]
Well... the glass is going to have to be of a size that can be moved around to get it to bottle banks so I'd suggest we want some way of making sure glass gets bagged up in strong plastic bags in a non-bulky way?

Hadn't thought about milk bottles. Yes they should go to the milko shouldn't they. How does the milk arrive in the morning? Wouldn't they normally expect us to leave empties wherever they leave the new bottles?

**Comment by johnmckerrell** [comment]
Given there won't be much glass (usually) could you just take the plastic box home one night and then bring it back in the next day without bags?

I think if we have a party or something we can bag the glass up and just pay bulky bobs anyway. I guess potentially we could do that normally, just make sure they don't take (and charge us for) a half empty bag.

**Comment by ajlennon** [comment]
That could work...! I'm mainly keen to get away from this situation where we have empty beer and alco-pops stuffed into cubby holes. If it all goes into a plastic box to be taken away then great!

**Comment by amcewen** [comment]
Milk bottles should totally be left out for the milkman.  I think at the moment I'm the only person who remembers to do that, and I don't always remember.

We currently get milk on Monday, Wednesday and Friday, so obviously need to be leaving empties out on the previous evenings.

**Comment by zarino** [comment]
I’ve set up a box for collecting the Mortons Dairies glass milk bottles:

![img_7631](https://user-images.githubusercontent.com/739624/42449859-26c35984-837a-11e8-90ec-2391be0b08e0.JPG)

Let’s see whether the bottles there currently are _still_ there on Wednesday morning 😉

**Comment by DoESsean** [comment]
Seems like this system is working so far, can we close this issue?

**Comment by zarino** [comment]
@DoESsean Is there really a "system"? The red box I made was really only for storing milk bottles between deliveries/collections, but it looks like it’s now also being used to store random other glass bottles, which I’m not sure anyone is actually collecting/recycling? I could be wrong though!

**Comment by JackiePease** [comment]
We now use a small crate in the kitchen to collect glass bottles for recycling. This needs to be emptied periodically into the wheelie bin labelled “Bin for Glass”. This is usually in the yard, under the fire escape from Queensway’s container, although if it’s just been emptied it may be on the other side of Commune.

During Hack the Space day, I added a sign to the wall above the crate, and we labelled the wheelie bin, with @magman2112 learning how to use the vinyl cutter as part of the process.
